src/sentry/tasks/symbolication.py

```diff
@@ -106,7 +106,7 @@ def submit_symbolicate(
         cache_key=cache_key,
         start_time=start_time,
         event_id=event_id,
-        data=data,
+        data=None,
```

Comment:
@klboke How do you guarantee that `data` is present in `event_processing_store`? There is a call to `backup_unprocessed_event` just before `submit_symbolicate`, but it only writes to the store conditionally.

https://github.com/getsentry/sentry/blob/d68f433de4e06d91c563605cc81d22c7869112ba/src/sentry/tasks/store.py#L162-L165

Reply:
Thanks, I missed that.